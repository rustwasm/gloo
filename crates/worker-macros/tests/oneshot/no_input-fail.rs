#[gloo::worker::oneshot::oneshot]
async fn Worker() -> u32 {
    0
}

fn main() {}
