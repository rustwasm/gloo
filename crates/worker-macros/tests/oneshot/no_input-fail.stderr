error: expected 1 argument
 --> tests/oneshot/no_input-fail.rs:2:10
  |
2 | async fn Worker() -> u32 {
  |          ^^^^^^
